test/expected/partials_path_regex.js

```diff
@@ -1,14 +1,15 @@
 this["JST"] = this["JST"] || {};
 
 Handlebars.registerPartial("partial", Handlebars.template({"compiler":[6,">= 2.0.0-beta.1"],"main":function(depth0,helpers,partials,data) {
```

Comment:
@sindresorhus I'm totally agree. so, have you any plan to improve these test cases?